api/types/container/change_types.go

```diff
@@ -0,0 +1,23 @@
+package container
+
+const (
+	// ChangeModify represents the modify operation.
+	ChangeModify ChangeType = 0
+	// ChangeAdd represents the add operation.
+	ChangeAdd ChangeType = 1
+	// ChangeDelete represents the delete operation.
+	ChangeDelete ChangeType = 2
+)
+
+func (ct ChangeType) String() string {
+	switch ct {
+	case ChangeModify:
+		return "C"
+	case ChangeAdd:
+		return "A"
+	case ChangeDelete:
+		return "D"
+	default:
+		return ""
```

Comment:
Should we do something different here to make it more obvious when it's an unknown value?

Reply:
I was considering that; for now I stuck with what the CLI already used; https://github.com/moby/moby/issues/45235#issue-1647516498

And wasn't sure what else to put here (`"-"` ? `"?"` ? `<unknown>` ?) WDYT?

Reply:
I personally like something really really verbose that'll definitely screw up formatting like `<unknown>` or `<invalid>` because it increases the chances that someone notices (because it's definitely an error _somewhere_ if we run into that case) :sweat_smile:

Reply:
I guess the alternative _could be_ to 

- not implement the stringer interface, and instead make this a `Change()` with a `(string, error)` return.
- return `strconv.Itoa()` to return the numeric value

I do like the stringer interface though (is handy to have), and if we return an error, we also need to take into account "what if we add more options in future?" So yeah, perhaps `<unknown>` is the way to go 🤔